virtusize-auth/src/main/java/com/virtusize/android/auth/data/FacebookUser.kt

```diff
@@ -0,0 +1,18 @@
+package com.virtusize.android.auth.data
+
+/**
+ * A class representing a Facebook user. It inherits from [VirtusizeUser].
+ * @property firstName The user's first name.
+ * @property lastName The user's last name.
+ * @property email The user's email.
+ */
+data class FacebookUser(
```

Comment:
Sorry, I might have missed this in the past. Restrict the visibility to prevent the client from accessing it unnecessarily. 
```suggestion
internal data class FacebookUser(
```